### PR TITLE
read rebase logs from database instead of from the chain

### PR DESCRIPTION
**Pull request by sparrowDom**
- fetches rebase logs form the database instead of the chain
- does half as many rebase log requests

**issue:** https://github.com/OriginProtocol/ousd-analytics/issues/254

### CODE REVIEW

File: eagleproject/core/blockchain/harvest/transactions.py [hunk 1]
```diff
@@ -137,6 +138,27 @@ def explode_log_data(value):
         out.append(int(value[2 + i * 64 : 2 + i * 64 + 64], 16) / 1e18)
     return out
 
+# get rebase log at block number
+def get_rebase_log(block_number):
+    rebase_log = Log.objects.filter(
+        topic_0='0x09516ecf4a8a86e59780a9befc6dee948bc9e60a36e3be68d31ea817ee8d2c80',
```
Comment (shahthepro) [inline eagleproject/core/blockchain/harvest/transactions.py:144]: Perhaps, move this to [this file](https://github.com/OriginProtocol/ousd-analytics/blob/master/eagleproject/core/blockchain/sigs.py) as a constant?

**Review by shahthepro — APPROVED** [review]

File: eagleproject/core/blockchain/harvest/transactions.py (hunk 1 above)
Comment (sparrowDom) [reply eagleproject/core/blockchain/harvest/transactions.py:144]: Yes! done, thanks

File: eagleproject/core/blockchain/harvest/transactions.py [hunk 2]
```diff
@@ -137,6 +138,27 @@ def explode_log_data(value):
...
+    ).order_by('-block_number')[:1].get()
+
+    return rebase_log
+
+# get rebasing credits per token log at block number
+def get_rebasing_credits_per_token(block_number):
+    rebase_log = get_rebase_log(block_number)
+    explode_log_data(rebase_log.data)
+
+    credits_per_token = explode_log_data(rebase_log.data)[2]
```
Comment (mikeshultz) [inline eagleproject/core/blockchain/harvest/transactions.py:155]: 💥

Comment (sparrowDom) [reply eagleproject/core/blockchain/harvest/transactions.py:155]: some of our code is so dramatic 🙃

**Pull request merged**